airflow-core/src/airflow/models/dagbag.py

```diff
@@ -45,6 +45,7 @@
     AirflowClusterPolicyError,
     AirflowClusterPolicySkipDag,
     AirflowClusterPolicyViolation,
+    AirflowDagCycleException,  # type: ignore[no-redef]
```

Comment:
Won't this cause it to use the old/deprecated name and issue a warning?

i.e. shouldn't this _always_ be `airflow.sdk.exceptions import AirflowDagCycleException` now?

Reply:
I left this because it's used in serialization and that could be factored out. Do you suggest a direct sdk import?